src/Filesystem/FileList.php

```diff
@@ -132,6 +132,14 @@ public function getEnvDistConfig(): string
         return $this->directoryList->getMagentoRoot() . '/.magento.env.md';
     }
 
+    /**
+     * @return string
+     */
+    public function getLogDistConfig(): string
```

Comment:
The naming should be aligned with the main config file.

Something like `.magento-log.env.yaml`

@YPyltiai

Reply:
@shiftedreality if this is just a file to contain text to merge with .magento.env.yaml - why would we care about name? It's technically not moving functionality from .magento.env.yaml AFAIK.

Reply:
Yes, it's just a file for storing log markdown, it won't be copied into the project root, only its content will be added to generated .magento.env.md file.